### ISSUE TITLE
when I use the pre-trained pose model to predict, the output is same whatever inputs are.

### ISSUE THREAD

**Issue by KerunLi** [tinghuiz/SfMLearner#136]
when I use the pre-trained pose model to predict, the output is same whatever inputs are.

I use seq 9 and seq 11 to run test_kitti_pose.py, but **the output pose.txt are all the same**, that is, 000000.txt and 000001.txt has the same content.

I print the input image data, it's not same, but the output pose is same. I normalized the input image data but it doesn't work.

（I use tensorflow-gpu 2.0.0b, cuda 10.0 and python 3.6, and use "`import tensorflow.compat.v1 as tf`" and "`tf.disable_v2_behavior()`" replace `import tensorflow as tf`

thanks a lot for any help.
